core/ops_json.rs

```diff
@@ -25,18 +25,18 @@ use std::rc::Rc;
 /// When registering an op like this...
```

Comment:
Maybe this module should be merged into core/ops.rs ? At least "json" should be removed from the name.

Reply:
Yeah, I was planning to rename this file in a separate "cleanup" pass that would also update the docstrings. Given that this PR already touches a lot of code I wanted to keep things separate.

Reply:
Leaving this for another PR